lecli/api_utils.py

```diff
@@ -308,6 +310,46 @@ def get_management_url():
         return 'https://rest.logentries.com/management'
 
 
+def get_query_url():
+    """
+    Get management url from the config file
+    """
+    config_key = 'query_url'
+    try:
+        url = CONFIG.get(URL_SECTION, config_key)
+        if validators.url(str(url)):
+            return url
+        else:
+            print_config_error_and_exit(URL_SECTION, 'Log Query URL(%s)' % config_key)
+    except (ConfigParser.NoOptionError, ConfigParser.NoSectionError):
+        return 'https://rest.logentries.com/query'
```

Comment:
This should be extracted as a constant. So it's easier to modify later. As well as the other default urls in the file